core/ir/riscv64/instr_create_api.h.in

```diff
@@ -120,16 +120,18 @@
  * \param d   The destination register opnd.
  * \param s   The source register opnd.
  */
-#define XINST_CREATE_move(dc, d, s) instr_create_1dst_1src(dc, OP_c_nop, d, s)
-
+#define XINST_CREATE_move(dc, d, s) \
+    INSTR_CREATE_addi(              \
+        dc, d, s,                   \
+        opnd_add_flags(opnd_create_immed_int(0, OPSZ_12b), DR_OPND_IMM_PRINT_DECIMAL))
 /**
  * This platform-independent macro creates an instr_t for a multimedia
  * register load instruction.
  * \param dc  The void * dcontext used to allocate memory for the instr_t.
  * \param r   The destination register opnd.
  * \param m   The source memory opnd.
  */
-#define XINST_CREATE_load_simd(dc, r, m) instr_create_1dst_1src(dc, OP_c_nop, r, m)
+#define XINST_CREATE_load_simd(dc, r, m) XINST_CREATE_load(dc, r, m)
```

Comment:
In this case what will be the opsz? Would XINST_CREATE_load really work?

Reply:
No, it does not work, RISC-V doesn't have any SIMD registers for now. Same as what's done in AArch64, I implement it as an `XINST_CREATE_load` for placeholding. Since it does not cause issues on the AArch64 side, I'm guessing this is not used by anyone.

(They're called in some functions in `core/ir/instr_shared.c`, but only executes when `reg_is_xmm(reg) || reg_is_mmx(reg)` is true, for ARM/RISC-V, that condition is always false.)